pages/pipelines/managing-priorities.md.erb

```diff
@@ -0,0 +1,49 @@
+# Job Prioritization
+
+Jobs are dispatched in the following order
+
+  - priority desc (highest priority -> lowest priority)
+  - scheduled_at asc (oldest jobs, note that jobs inherit "scheduled_at" from pipeline upload jobs, meaning jobs that are pipeline uploaded in an older build will be dispatched before later created builds)
+  - order asc (the order they were uploaded in a pipeline)
+  - internal id asc (used as a tie breaker if all value are the same, meaning older jobs will be dispatched first)
+
+Jobs default to a priority of 0, you can prioritise or deprioritise by using higher or lower numbers.
+
+<video src="https://user-images.githubusercontent.com/14028/122886342-39460a00-d383-11eb-898d-692aa057ffa0.mov"></video>
+
+Here's a very basic example:
+
+```yml
+steps:
+  - command: "will-run-last.sh"
+    priority: -1
+  - command: "will-run-first.sh"
+    priority: 1
+```
+{: codeblock-file="pipeline.yml"}
+
+Priority is considered across the entire organization when dispatching.
```

Comment:
Perhaps it's clearer to say across the queue (which is org wide)? We don't mention queues in the words here at all, aside from the code example for uploader, but queues imply ordering so it might be good to somehow explain things in terms of queueing (maybe with a diagram even?)

Reply:
Agree with `queue` or `pipeline` (which is it?)

Reply:
Yeah, interesting. Priority is technically implemented at an organization level, so within a single dispatch which might span multiple queues (as in the agent tag called 'queue') the jobs with higher priority will be assigned before jobs with lower priority.

Reply:
@sj26 would you mind taking a :eyes: at the priority/queues stuff especially?